.murdock

```diff
@@ -169,7 +169,7 @@ get_app_board_toolchain_pairs() {
 
 # use dwqc to create full "appdir board toolchain" compile job list
 get_compile_jobs() {
-    check_label "CI: skip compile test" && return
+    check_label "CI: skip compile test" && exit
```

Comment:
no, this needs to be return

Reply:
Still doesn't explain why it still collects all jobs... Local testing works with either.

Reply:
(though of course it results in everything after being skipped with `exit`)

Reply:
Grml... All but a very few environment variables are filtered out here https://github.com/RIOT-OS/murdock-scripts/blob/4904b5111b5c7394bc806c45ef7404d435b88f0e/common.sh#L40-L42